depmapomics/upload.py

```diff
@@ -0,0 +1,648 @@
+from __future__ import print_function
```

Comment:
let's move this in a separate repo

Reply:
Not sure if you meant to comment on some specific lines.. 1) yes this is going to be moved to a separate repo once its ready to be merged with Sarah's progress 2) on dev in ccle_tasks/gumbo_test.ipynb there are some function calls but not very well documented. I will work on that 3) I'm not sure what decisions you are referring to?

Reply:
TODO: add _profile and _model suffix to the upload functions; remove one_pr_per_type; rename file names on latest to agree w/ virtual